build.sh

```diff
@@ -52,11 +52,18 @@ mkdir -p ${BUILD_ARCH}_SCT/SCT
 cp -r Build/UefiSct/RELEASE_GCC5/SctPackage${BUILD_ARCH}/$BUILD_ARCH/* ${BUILD_ARCH}_SCT/SCT/
 cp Build/UefiSct/RELEASE_GCC5/SctPackage${BUILD_ARCH}/SctStartup.nsh ${BUILD_ARCH}_SCT/Startup.nsh
 
+# Copy the E/S BBR sequence files
+cp SBBR.seq ${BUILD_ARCH}_SCT/SCT/Sequence/
+cp EBBR.seq ${BUILD_ARCH}_SCT/SCT/Sequence/
```

Comment:
Instead of the seq files being linked into the base directory and copying from there, the files should simply be copied from the scripts directory.